x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/index_data_visualizer_view/index_data_visualizer_view.tsx

```diff
@@ -454,131 +448,154 @@ export const IndexDataVisualizerView: FC<IndexDataVisualizerViewProps> = (dataVi
-                  onAddFilter={onAddFilter}
-                />
-
-                {overallStats?.totalCount !== undefined && (
-                  <>
-                    <EuiSpacer size={'m'} />
-                    <EuiFlexItem grow={true}>
-                      <DocumentCountContent
-                        documentCountStats={documentCountStats}
-                        totalCount={overallStats.totalCount}
-                        setSamplingProbability={setSamplingProbability}
-                        samplingProbability={
-                          dataVisualizerListState.probability === null
-                            ? documentCountStats?.probability
-                            : dataVisualizerListState.probability
-                        }
-                        loading={overallStatsProgress.loaded < 100}
-                        randomSamplerPreference={savedRandomSamplerPreference}
-                        setRandomSamplerPreference={saveRandomSamplerPreference}
-                      />
-                    </EuiFlexItem>
-                  </>
-                )}
-                <EuiSpacer size={'m'} />
-                <FieldCountPanel
-                  showEmptyFields={showEmptyFields}
-                  toggleShowEmptyFields={toggleShowEmptyFields}
-                  fieldsCountStats={fieldsCountStats}
-                  metricsStats={metricsStats}
+              ) : null}
+              <EuiFlexItem grow={false}>
+                <DatePickerWrapper
+                  isAutoRefreshOnly={!hasValidTimeField}
+                  showRefresh={!hasValidTimeField}
+                  compact={compact}
                 />
-                <EuiSpacer size={'m'} />
-                <EuiProgress value={progress} max={100} size={'xs'} />
-                <DataVisualizerTable<FieldVisConfig>
-                  items={configs}
-                  pageState={dataVisualizerListState}
-                  updatePageState={setDataVisualizerListState}
-                  getItemIdToExpandedRowMap={getItemIdToExpandedRowMap}
-                  extendedColumns={extendedColumns}
-                  loading={progress < 100}
-                  overallStatsRunning={overallStatsProgress.isRunning}
-                  showPreviewByDefault={dataVisualizerListState.showDistributions ?? true}
-                  onChange={setDataVisualizerListState}
-                  totalCount={overallStats.totalCount}
-                />
-              </EuiPanel>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false} style={{ width: wizardPanelWidth }}>
-              <ActionsPanel
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiPageContentHeader>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="m" />
+      <EuiPageContentBody>
+        <EuiFlexGroup
+          gutterSize="m"
+          css={
+            compact &&
+            css`
+              flex-direction: column;
+            `
+          }
```

Comment:
if I'm not mistaken there is a dedicated prop for `direction`

Reply:
Updated here [`38b8a06` (#147137)](https://github.com/elastic/kibana/pull/147137/commits/38b8a0645d85f782fd329c139d23280396cbcaa9)